maketeams3.py

```diff
@@ -23,14 +27,18 @@ def __init__(self, name, rating, friends, avoid, date, alt, previous_season_alt)
 
     @classmethod
     def player_from_json(cls, player):
+        if 'previous_season_alternate' in player:
+            prev_alt = player['previous_season_alternate'] == 'alternate'
+        else:
+            prev_alt = False
         return cls(
             player['name'],
             player['rating'],
             player['friends'],
             player['avoid'],
             player['date_created'],
             player['prefers_alt'],
-            player['previous_season_alternate'] == 'alternate'
+            prev_alt
```

Comment:
I don't really see why we need this kind of defense since all the values are coming from the website anyway so should be set. Or why not have this check for all fields? I know that the prev_alt is not from the registration form but rather from the site but it should still be in the export. Either way - would prefer to use dictionary get() to do this rather than the if else. Thoughts welcome.

Reply:
I did this because the field doesn't exist in s14.  I also don't understand how the dictionary get method would work here.  It returns the value or None right?  We're looking for True or False.

Reply:
`player.get('previous_season_alternate') == 'alternate'` will work. It will return True of the key exists and contains `'alternate'` It will return False otherwise

Reply:
Ah, I get it now.  That's clever.